config/groovy/GooeyCLI.groovy

```diff
@@ -0,0 +1,39 @@
+// Copyright 2020 The Terasology Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+// Grab the Groovy extensions for PicoCLI - in IntelliJ Alt-ENTER on a `@Grab` to register contents for syntax highlighting
```

Comment:
Ideally lines 4-12 here would disappear if I can use the code from inside the Gradle Wrapper instead